tensorrt_llm/_torch/disaggregation/native/transfer.py

```
@@ -738,6 +799,22 @@ def clear_session(self, unique_rid: int):
+        req_info_map = self._get_req_info(unique_rid)
+        if not req_info_map:
+            return
+        for req_info in req_info_map.values():
+            try:
+                peer_ri = self._registrar.get_peer_rank_info(
+                    req_info.instance_name, req_info.instance_rank
+                )
+                self._get_or_connect_dealer(peer_ri.self_endpoint).send(
+                    [MessageType.CANCEL_SESSION, str(unique_rid).encode("ascii")]
+                )
+            except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)